src/guide/essentials/list.md

```diff
@@ -134,7 +134,7 @@ You can also use `of` as the delimiter instead of `in`, so that it is closer to
 
 ## `v-for` with an Object {#v-for-with-an-object}
 
-You can also use `v-for` to iterate through the properties of an object. The iteration order will be based on the result of calling `Object.keys()` on the object:
+You can also use `v-for` to iterate through the properties of an object. The iteration order will be based on the result of calling `Object.values()` on the object:
```

Comment:
The keyword here is "iteration order" and as [AloisSeckar](https://github.com/AloisSeckar) wrote above, both return elements in same order. What you get in `v-for="_____ in obj"` is different thing and is shown by the example just below this. Also, `Object.keys()` is the thing we internally use.

Reply:
ok - makes sense! Thanks for explaining and Thanks for having such great documentation!